ci/build-gmt-master.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+#
+# Build the lastest GMT source codes
+#
+
+set -x -e
+
+# Following variables can be modified via environment variables
+GMT_INSTALL_DIR=${GMT_INSTALL_DIR:-${HOME}/gmt-install-dir}
+
+# General settings
+GSHHG_VERSION="2.3.7"
+DCW_VERSION="1.1.4"
+GSHHG="gshhg-gmt-${GSHHG_VERSION}"
+DCW="dcw-gmt-${DCW_VERSION}"
+EXT="tar.gz"
+
+cwd=${PWD}
+# 1. Create temporary directory for building
+rm -rf /tmp/build-gmt
+mkdir -p /tmp/build-gmt
+cd /tmp/build-gmt
```

Comment:
There is a program called `mktemp` to do this and avoid deleting something that we shouldn't have. Not sure how common it is in Linux distros, though. It seems to be common since it's on the Ubuntu [GNU Core Utils package](https://launchpad.net/ubuntu/trusty/+package/coreutils). So it might be a safe bet that it will be present.